src/internal/components/drag-handle/index.tsx

```diff
@@ -13,7 +12,9 @@ import styles from './styles.css.js';
 export interface DragHandleProps {
   attributes: ButtonHTMLAttributes<HTMLDivElement>;
   hideFocus?: boolean;
-  listeners?: SyntheticListenerMap;
+  // @dnd-kit uses this type
+  // eslint-disable-next-line @typescript-eslint/ban-types
+  listeners: Record<string, Function> | undefined;
```

Comment:
yes, this is the whole type

https://github.com/clauderic/dnd-kit/blob/e2a1776d0de657669192d3cfd1558e91905b5fad/packages/core/src/hooks/utilities/useSyntheticListeners.ts#L12

This is not something that is worth reusing